File: samples_generated_code/samples_2/phi4/baseline_code_c/code_row_68.c
#include <stdio.h>
#include <stdlib.h>
#include <time.h>

int main() {
    srand(time(NULL));
    int randomNumber = rand() % (9999 - 1000 + 1) + 1000;
    printf("Random number larger than 1000: %d\n", randomNumber);
    return 0;
}

File: samples_generated_code/samples_2/phi4/baseline_code_java/code_row_68.java
import java.util.Random;

public class RandomNumberGenerator {
    public static void main(String[] args) {
        Random rand = new Random();
        int randomNumber = rand.nextInt(9000000) + 1001; // Generates a random number between 1001 and 9000000
        System.out.println("Random Number: " + randomNumber);
    }
}